docs/native-modules-android.md

```diff
@@ -417,12 +621,31 @@ const onPress = () => {
 
 Another option is to use an onSuccess and onFailure callback:
 
+<Tabs groupId="android-language" defaultValue={constants.defaultAndroidLanguage} values={constants.androidLanguages}>
+<TabItem value="java">
+
 ```java
 @ReactMethod
 public void createCalendarEvent(String name, String location, Callback myFailureCallback, Callback mySuccessCallback) {
 }
 ```
 
+</TabItem>
+<TabItem value="kotlin">
+
+```kotlin
+@ReactMethod
```

Comment:
That's wrongly aligned? All the code should be formatted with 
https://facebookincubator.github.io/ktfmt/